crates/db/src/traits.rs

```diff
@@ -177,7 +182,7 @@ pub enum BlockStatus {
 /// For now, the full state is just the "toplevel" state that can always be
 /// expected to be of moderate size in memory.
 // TODO maybe rewrite this around storing write batches according to blkid?
-pub trait ChainstateDatabase {
+pub trait OldChainstateDatabase {
```

Comment:
We should remove this trait and all the stuff built on top of it, since we have supersceded it.